CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
 ## [Unreleased]
+- Fix creating translations with app - #6804 by @krzysztofwolski
```

Comment:
Please move this entry to the "Other" section below, this is where we keep all entries that are not breaking changes.

Reply:
Done